code/modules/shuttle/supply.dm

```diff
@@ -346,8 +346,8 @@
 	slip.info +="CONTENTS:<br><ul>"
 
 	//we now create the actual contents
-	var/list/contains
-	if(istype(object, /datum/supply_packs/misc/randomised))
+	var/list/contains // TODO: de-snowflake-ify randomisation
+	if(istype(object, /datum/supply_packs/misc/randomised) || istype(object, /datum/supply_packs/engineering/randomised/powergamermitts))
```

Comment:
You're better off just making a type of a glove that can spawn as either a real one or a fake one and using it here, rather than doing....this.